roles/cifmw_cephadm/tasks/pools.yml

```diff
@@ -47,19 +47,21 @@
 
 - name: Configure the RBD trash purge scheduler
   when:
-    - cifmw_enabled_services | default([]) | intersect(['cinder_volume'])
     - cifmw_cephadm_enable_trash_scheduler | default(false)
```

Comment:
If we introduce `cifmw_cephadm_pools_with_trash_scheduler` as a list, then we could get rid of the `cifmw_cephadm_enable_trash_scheduler` boolean remove the `when` and then loop on it.

We could add an additional condition in the when to ensure an intersection with `cifmw_cephadm_enable_trash_scheduler` and `cifmw_cephadm_pools` but that might be unnecessary and lead to further unnecessary complication in the future.

Reply:
Let's go with the tag in the original `cifmw_cephadm_pools` instead.